src/Opdex.Platform.Application/Handlers/Indexer/MakeIndexerUnlockCommandHandler.cs

```diff
@@ -8,26 +7,17 @@
 
 namespace Opdex.Platform.Application.Handlers.Indexer;
 
-public class MakeIndexerUnlockCommandHandler : IRequestHandler<MakeIndexerUnlockCommand, Unit>
+public class MakeIndexerUnlockCommandHandler : AsyncRequestHandler<MakeIndexerUnlockCommand>
```

Comment:
Doesn't really matter but you could stick with IRequestHandler here and return `Unit.Value` instead 🤷‍♂️